content/en/blog/_posts/2021-04-29-SIG-Usability-Spotlight.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: blog
+title: "SIG-Usability Spotlight"
+date: 2021-04-29
+slug: sig-usability-spotlight-2021
+---
+
+**Author:** [Kunal Kushwaha](https://twitter.com/kunalstwt)
+
+## Introduction
+
+Are you interested in learning about what Sig-Usability does and how you can get involved? Well, you're at the right place. Sig-Usability is all about making Kubernetes more accessible to new folks, and its main activity is conducting user research for the community. In this blog, we have summarized our conversation with [Gaby Moreno](https://twitter.com/morengab) who walks us through the various aspects of being a part of the SIG and shares some insights about how others can get involved.
```

Comment:
Also, is there a resource you can link the first occurrence of "SIG Usability" to?

Reply:
Thanks for the comments, changes made